external/include/protocols/ftpc.h

```diff
@@ -88,7 +88,7 @@
 #endif
 
 #ifndef CONFIG_FTP_TMPDIR
-#define CONFIG_FTP_TMPDIR "/tmp"
```

Comment:
In order to use `/tmp` dir following configs should be enabled, but these are disbaled in `artik053/nettest`. 
```
CONFIG_FS_TMPFS=y
CONFIG_FS_AUTOMOUNT_TMPFS=y
```
So can you please verify by enabling above 2 configs instead of replacing `/tmp` with `/mnt` dir